ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -128,11 +130,28 @@ spec:
 \${UPGRADE_ICS}
 EOF1
 
-echo "Mirroring release images for disconnected environment"
-\$MIRRORCOMMAND
+#Retry logic for the real release mirror command
```

Comment:
retry logic would be good, but you might want to delay until after you've confirmed that you've fixed fail-fast error-propagation, just to simplify what you need to hold in your head when resolving error-propagation.  Or are you confident you've already fixed the error-propagation issue?  I see your pull still has a `[WIP]` title prefix.

Reply:
@wking from the recent tests I've done it looks like  moving the code from L134 to L192 did the trick, it was interfering somehow with the output of the function
I believe we can leave the retry logic as is now, but please let me know if you have any more comments on this